src/index.ts

```diff
@@ -177,6 +189,11 @@ if (argv) {
     let silence = argv.json || argv.quiet || argv.xml ? true : false;
     let artie = argv.artie ? true : false;
     let allen = argv.allen ? true : false;
+    let directory = argv.directory ? argv.directory : process.cwd();
+
+    if (directory && directory !== process.cwd()) {
+      console.log('Running on directory', directory);
```

Comment:
Nice touch!

Reply:
SUGGESTION: Rather than use `console.log`, use: logMessage('', DEBUG); at whatever level you'd like to write it to the log.